devtools/replica-setup-pg/README.md

```diff
@@ -1,4 +1,4 @@
-# PostgreSQL to MyDuck Replication Setup
```

Comment:
The directory name and related description should be updated to `load-from-pgdump` for better clarity. The term "Replica" implies that MyDuck Server will continuously synchronize with a source database (e.g., MySQL or Postgres) by receiving both snapshot and delta updates. However, this document specifically describes the process of loading a `pg_dump` file into MyDuck Server, which is a distinct use case. Renaming the directory will help differentiate these functionalities.